frontend/packages/db-structure/src/schema/overrideDbStructure.ts

```diff
@@ -44,10 +51,12 @@ export type DBOverride = v.InferOutput<typeof dbOverrideSchema>
 /**
  * Applies override definitions to the existing DB structure.
  * This function will:
- * 1. Apply overrides to existing tables (e.g., replacing comments)
- * 2. Add new columns to existing tables
- * 3. Add new relationships
- * 4. Process and merge table groups from both original structure and overrides
+ * 1. Apply overrides to existing tables
+ * 1.1. Replace comments
+ * 1.2. Add new columns
+ * 1.3. Add new constraints
+ * 2. Add new relationships
+ * 3. Process and merge table groups from both original structure and overrides
```

Comment:
I arranged it a little because I think "adding new columns" step is a part of "applying overrides to existing tables". Does that align with your intention?

Reply:
Sorry, we now have a policy of not unnecessarily adding more information to overrideDbStructure. Can you turn it off now?

- ref: https://github.com/liam-hq/liam/pull/1194

Reply:
I removed the lines to merge `constraints` from the function. It becomes much simpler, thanks 😉 
Bad timing tho.

I fixed it in these commits: https://github.com/liam-hq/liam/pull/1168/commits/1b5c44ee868554fc7f84a930fa8df21bf8eb3621 (revert to the current main), https://github.com/liam-hq/liam/pull/1168/commits/668fd7708b542fb0359072024a6e47840736955f (fix test)